10.9-libcxx/stable/main/finkinfo/sci/numexpr-py.info

```diff
@@ -1,23 +1,31 @@
 Info2: <<
 Package: numexpr-py%type_pkg[python]
-Version: 2.6.4
-Revision: 2
+Version: 2.6.8
+Revision: 1
 Maintainer: Kurt Schwehr <goatbar@users.sourceforge.net>
-Type: python (2.7 3.4 3.5 3.6)
+Type: python (2.7 3.4 3.5 3.6 3.7)
 Depends: <<
 	python%type_pkg[python],
 	numpy-py%type_pkg[python] (>= 1.14.5-2)
 <<
 BuildDepends: fink (>= 0.24.12)
-Source: https://pypi.python.org/packages/74/58/466cfe707a78e6c11725c7be9cee71042bd902fe175bd67429dba655bbd9/numexpr-%v.tar.gz
-Source-MD5: 3898ff5187e6d4c31b0c0f7f677d21d8
+Source: https://files.pythonhosted.org/packages/source/n/numexpr/numexpr-%v.tar.gz
+Source-MD5: 8148ba96476e2c367292789bc2a90d85
 
 GCC: 4.0
 CompileScript: <<
- %p/bin/python%type_raw[python] setup.py build
+ SHELL=/bin/sh %p/bin/python%type_raw[python] setup.py build
```

Comment:
That should have been fixed by #121 (i.e. the `numpy-py%type_pkg[python] (>= 1.14.5-2)` dep).

Reply:
Is the 1.15.1 update something you are working on? Preferably the patch to disable `sh = os.environ.get('SHELL', '/bin/sh')` in numpy should be kept working, because there are other packages that depend on it now.

Reply:
I had a first go at it, but there are quite a number of test failures and i am not sure, if i can fix all of that. Assertion errors, rank warnings, and more such nasty stuff. I did not expect this from such a central python module.

Reply:
I will probably leave it and move on to matplotlib, which has a much larger version lag, although the first test on this was as bad.

Reply:
I did have a test build of `numpy-py-1.15.0` with a bunch of test failures after the switch to `pytest`, but for the most part looking non-critical - rather missing dependencies on a load of new `pytest-helper` packages, unfortunately.
I had also been looking into the newer matplotlib versions (2.x, not 3.0.0 yet); should look at the status of that. Sorry I slipped a bit out of the loop after travelling this summer...

Reply:
I have pytest-cov basically ready. Do you remember, what else?